baseband/vdif/header.py

```diff
@@ -671,10 +665,11 @@ def get_time(self, framerate=None, frame_nr=None):
             if frame_nr == 0:
                 offset = 0.
             else:
-                if framerate is None:
-                    raise ValueError("calculating the time for a non-zero "
-                                     "frame number requires a frame rate. "
+                if sample_rate is None:
+                    raise ValueError("Calculating the time for a non-zero "
+                                     "frame number requires a sample rate. "
                                      "Pass it in explicitly.")
+                framerate = sample_rate / self.samples_per_frame
                 offset = (frame_nr / framerate).to(u.s).value
```

Comment:
I think I'd do (here and elsewhere) `offset = (frame_nr * self.samples_per_frame / sample_rate).to(u.s).value`, but it doesn't really matter.